ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphContainer.tsx

```diff
@@ -313,12 +323,15 @@ function NetworkGraphContainer({
     let filteredNodes: CustomNodeModel[] = [...baseModel.nodes];
     let filteredEdges: CustomEdgeModel[] = [...baseModel.edges];
     // if edgeState is extraneous && there is a selectedNode, add in/egress flows nodes/edges
-    if (edgeState === 'inactive' && selectedNode?.data.type === 'DEPLOYMENT') {
+    if (edgeState !== 'active' && selectedNode?.data.type === 'DEPLOYMENT') {
```

Comment:
Did you mean to flip the check here?

Before it had to be "inactive", but now it just has to be something other than "active" ?

Reply:
Yep, I wanted to flip it. Before we had just `active` and `inactive`. So the `edgeState === 'inactive'` meant that it was `inactive`. Now that we technically have two `inactive` states, one with DNS and one without, the `edgeState !== 'active'` encompasses both those states. If we ever, God forbid, add more states, we'll have to redo this logic again 😢